backend/logger/db_logger.py

```diff
@@ -0,0 +1,20 @@
+import mysql.connector
+from mysql.connector import Error
+import pandas as pd
+
+
+def create_server_connection(host_name, user_name, user_password):
+    connection = None
+    try:
+        connection = mysql.connector.connect(
+            host=host_name, user=user_name, passwd=user_password
+        )
+        print("MySQL Database connection successful")
+    except Error as err:
+        print(f"Error: '{err}'")
+
+    return connection
+
+
+if __name__ == "__main__":
+    connection = create_server_connection("localhost", "root", pw)
```

Comment:
I can add instructions for using this to the README. In terms of the choice of db or local logging, I'm not sure. That is a design choice we could make as a team. Since this is only used by developers and not the end-users (i.e. the students and teachers), I don't see a point of decision making at runtime. I can provide a DB logger that is separate, and developers can use independently. @slakh96  @sultansidhu  What do you think?

Reply:
sure, sounds good to me, adding README instructions and/or giving a demo during meeting would be great